stack.yaml

```diff
@@ -116,10 +119,10 @@ packages:
 
 # Because log-warper mainline depends on a lifted-async which depends upon a
 # buggy async
-- location:
-    git: https://github.com/input-output-hk/log-warper.git
-    commit: fa925f9dadf239b38e2ffcc35f9d241a9fcb7d55
-  extra-dep: true
+# - location:
+#    git: https://github.com/input-output-hk/log-warper.git
+#    commit: 5271ab6c33541b8155ca203e714875974ec116be
+#  extra-dep: true
```

Comment:
Delete this lines? Are you sure it's OK to use the stackage/hackage version?

Reply:
@avieth I'll probably have to look at the diff a bit closer, I was mostly concerned with getting this to build at all for now.